source/accessories/amds/mainboard/index.md

```diff
@@ -146,8 +146,8 @@ One header is used to supply power to the cards and the other header is used for
 | 2 | 3V3 |
 | 3 | GND |
 | 4 | DIN (ADC IN, MOSI) |
-| 5 | DOUT (ADC OUT, MISO) |
-| 6 | SCLK |
+| 5 | SCLK |
+| 6 | DOUT (ADC OUT, MISO) |
```

Comment:
@Known4225, did you mean to edit these lines?

Reply:
yes, this is to close #111 and I can confirm from both schematic and physical testing that this is the correct order